compatibility/bazel_tools/daml_ledger/Main.hs

```diff
@@ -249,3 +283,16 @@ writeMinimalProject (SdkVersion sdkVersion) = do
     , "module Main where"
     , "template T with p : Party where signatory p"
     ]
+
+supportsNoBearerPrefix :: SdkVersion -> Bool
+supportsNoBearerPrefix ver =
+    ver >= SdkVersion
+        ( SemVer.initial
+        & SemVer.major .~ 1
+        & SemVer.minor .~ 1
+        & SemVer.patch .~ 1
+        )
+
+-- | TODO bump to the first snapshot after 1.4.0-snapshot.20200715.4733.0.d6e58626.
+supportsTimeout :: SdkVersion -> Bool
+supportsTimeout ver = ver >= SdkVersion SemVer.initial
```

Comment:
Can't you just say `>` and avoid having to remember this next week?

Reply:
No 1.4.0 is not > 0.0.0

Reply:
Oh you mean the current snapshot, yeah that works. I’ll change it.

Reply:
Well of course it isn't, I meant `> 1.4.0-snapshot.20200715.4733.0.d6e58626`.

Reply:
changed.